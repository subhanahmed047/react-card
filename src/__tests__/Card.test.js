import React from 'react';
import { create } from 'react-test-renderer';
import CardWrapper from '../components/card/wrapper';

describe('Card Wrapper component', () => {
    test('card wrapper component matches the snapshot', () => {
        const component = create(<CardWrapper />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
