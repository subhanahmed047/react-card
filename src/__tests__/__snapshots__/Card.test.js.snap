// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Wrapper component card wrapper component matches the snapshot 1`] = `
<div
  className="sc-htpNat eyZxIT"
/>
`;
